src/Microsoft.Performance.SDK/Processing/ColumnConfiguration.cs

```diff
@@ -39,10 +41,34 @@ public ColumnConfiguration(ColumnMetadata metadata)
         ///     <paramref name="metadata"/> is <c>null</c>.
         /// </exception>
         public ColumnConfiguration(ColumnMetadata metadata, UIHints hints)
+            : this(metadata, hints, null)
+        {
+        }
+
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="ColumnConfiguration"/>
+        ///     class.
+        /// </summary>
+        /// <param name="metadata">
+        ///     The metadata about the column.
+        /// </param>
+        /// <param name="hints">
+        ///     Optional hints about displaying this column in the UI. This parameter
+        ///     may be <c>null</c>. If this parameter is <c>null</c>, then
+        ///     <see cref="UIHints.Default"/> will be used for this instance.
+        /// </param>
+        /// <param name="variantGuid">
+        ///     Optional unique identifier of the column variant to use.
+        /// </param>
+        /// <exception cref="System.ArgumentNullException">
+        ///     <paramref name="metadata"/> is <c>null</c>.
+        /// </exception>
+        public ColumnConfiguration(ColumnMetadata metadata, UIHints hints, Guid? variantGuid)
```

Comment:
Consolidate the ctors with optional parameters?

Reply:
Unfortunately I cannot add or modify the previous constructors because we need to maintain backwards compatibility

Reply:
I believe having optional parameters still makes it backward compatible. They can still call it with 2 parameters and the third parameter will use the default value

Reply:
Done!